lib/pacemaker/pcmk_sched_transition.c

```diff
@@ -849,7 +846,7 @@ run_simulation(pe_working_set_t * data_set, cib_t *cib, GList *op_fail_list)
     }
 
     if (graph_rc != transition_complete) {
-        return graph_rc;
+        return -graph_rc;
     }
-    return 0;
+    return pcmk_rc_ok;
```

Comment:
run_simulation() doesn't return a standard or legacy code, it returns enum transition_status on success (0 == transition_active) and -1 on error. So, we weren't exactly using it correctly in crm_simulate, but it worked more or less coincidentally.

It's internal, so we can change it ... we should return the enum explicitly rather than int, return graph_rc without modification, and have the caller compare the result to transition_complete

Reply:
Yeah, that's much better.  Fixed.